kaggle_environments/envs/werewolf/visualizer/default/src/legacy-renderer.js

```diff
@@ -50,6 +50,9 @@ export function renderer(context, parent) {
   const environment = replay;
   const parentId = parent.id;
 
+  let playerNamesFor3D = [];
```

Comment:
@dster2 - the error you were seeing was if the ThreeModules promise finished before the other initialization code got to where these are defined - hoisting them up to the top level I'm hoping solves that race.